app/models/rule.rb

```diff
@@ -80,7 +80,9 @@ def canonical?
     (profiles & Profile.canonical).any?
   end
 
-  def self.from_openscap_parser(op_rule, existing: nil, rule_group_id: nil, benchmark_id: nil, precedence: nil)
+  # rubocop:disable Metrics/ParameterLists
+  def self.from_openscap_parser(op_rule, existing: nil, rule_group_id: nil,
+                                benchmark_id: nil, precedence: nil, value_checks: nil)
```

Comment:
@skateman Is it okay to disable this rubocop check for this method? We need to pass `value_checks` into this method but it brought the parameter count to 6 which rubocop complains about 🙁

Reply:
Yeah, let's keep it like that